components/core/src/testing/ui-test-configuration.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ElementRef, InjectionToken } from '@angular/core';
+import { coerceElement } from '@angular/cdk/coercion';
+
+/** Interface for Injectiontoken to set UI-test attribute to overlay Container */
+export interface DtUiTestConfiguration {
+  attributeName: string;
+  constructOverlayAttributeValue(uiTestId: string, id: number): string;
+}
+
+/** Default configuration for setting the UI-test attribute */
+export const DT_DEFAULT_UI_TEST_CONFIG: DtUiTestConfiguration = {
+  attributeName: 'uitestid',
+  constructOverlayAttributeValue(uiTestId: string, id: number): string {
```

Comment:
would default id to 0 
```suggestion
  constructOverlayAttributeValue(uiTestId: string, id = 0): string {
```